servers/services/src/test/java/org/projectnessie/services/rest/InstantParamConverterProviderTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.projectnessie.services.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+
+import javax.ws.rs.ext.ParamConverter;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class InstantParamConverterProviderTest {
```

Comment:
nit: why isn't the javadoc checkstyle arg picking these up? Most of the nessie codebase uses package private tests and test methods or adds javadocs for the tests

Reply:
I haven't checked, but my assumption was that Checkstyle isn't complaining on test classes. I was comparing with a few other test classes, such as `public class RestGitTest`